src/views/Events/index.js

```diff
@@ -0,0 +1,202 @@
+                    <Checkbox
+                      checked={this.state.academics}
+                      onChange={this.filterEvents('academics')}
+                    />
+                }
+                  label="Academics"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.admissions}
+                      onChange={this.filterEvents('admissions')}
+                    />
+                 }
+                  label="Admissions"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.art}
+                      onChange={this.filterEvents('art')}
+                    />
+                 }
+                  label="Arts"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.calendar}
+                      onChange={this.filterEvents('calendar')}
+                    />
+                 }
+                  label="Calendar Events"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.cec}
+                      onChange={this.filterEvents('cec')}
+                    />
+                 }
+                  label="CEC"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.chapelOffice}
+                      onChange={this.filterEvents('chapelOffice')}
+                    />
+                 }
+                  label="Chapel Office"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.studentLife}
+                      onChange={this.filterEvents('studentLife')}
+                    />
+                 }
+                  label="Student Life"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.fair}
+                      onChange={this.filterEvents('fair')}
+                    />
+                 }
+                  label="Fair or Expos"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.sports}
+                      onChange={this.filterEvents('Sports')}
+                    />
+                 }
+                  label="Athletics"
+                />
+              </FormGroup>
+              <Divider light />
+              <FormGroup row>
+                <FormControlLabel
+                  control={
+                    <Switch
+                      checked={this.state.chapelCredits}
```

Comment:
Why is this label different from the rest? It doesn't seem to work as the others do.

Reply:
that was a database thing it is good now